.vib/redmine/cypress/cypress/integration/redmine_spec.js

```diff
@@ -0,0 +1,86 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+it('allows admin to login/logout', () => {
+  cy.login();
+  cy.get('#flash_error').should('not.exist');
+  cy.contains('Sign out').click();
+  cy.contains('Sign in');
+});
+
+it('allows user to register', () => {
+  cy.visit('account/register');
```

Comment:
You're interchangeably using `cy.visit` with a leading `/` and without it. Use only one approach:

```suggestion
  cy.visit('/account/register');
```